external-crates/move/crates/move-analyzer/editors/code/README.md

```diff
@@ -101,8 +102,18 @@ Move source file (a file with a `.move` file extension) and:
   - inlay hints:
     - types: local declarations, lambda parameters, variant and struct pattern matching
     - parameter names at function calls
-- If the opened Move source file is located within a buildable project you can build and (locally)
+- If the opened Move source file is located within a buildable project, and you have the `sui`
+  binary installed, you can build and (locally)
   test this project using `Move: Build a Move package` and `Move: Test a Move package` commands from
-  VSCode's command palette. You can also enable Move trace generation during test execution
-  using `Move: Trace Move package execution` command from VSCode's command palette (traces will
-  be available in the `traces` directory in JSON format).
+  VSCode's command palette.
+- If the opened Move source file is located within a buildable project, and you have the `sui`
+  binary installed, you can trace-debug execution of Move unit tests within this project.
+  This functionality is provided by automatically including Move Trace Debugging
+  [extension](https://marketplace.visualstudio.com/items?itemName=mysten.move-trace-debuggin)
+  where you can find more detailed information about trace-debugging and the current level of support.
```

Comment:
This sentence is confusing to me, and I'm not sure it's saying this is automatically included in the larger Move IDE VSCode extension, or included in the move-trace-debugging extension (which is/is not automatically included with the IDE?). 

I think rewording/clarifying this may be helpful :)

Reply:
Done!